adcc/ElectronicTransition.py

```diff
@@ -214,25 +242,25 @@ def oscillator_strength_velocity(self):
                                self.excitation_energy)
         ])
 
-    @cached_property
     @mark_excitation_property()
-    def rotatory_strength(self):
+    def rotatory_strength(self, gauge_origin=(0.0, 0.0, 0.0)):
```

Comment:
Should it be possible to select a gauge origin here although the property is gauge-origin invariant?